components/feature/downloads/src/test/java/mozilla/components/feature/downloads/DownloadNotificationTest.kt

```diff
@@ -157,4 +158,28 @@ class DownloadNotificationTest {
         val summary = DownloadNotification.getSummaryList(testContext, listOf(download1, download2))
         assertEquals(listOf("mozilla.txt 10%", "mozilla2.txt 20%"), summary)
     }
+
+    @Test
+    fun getNotificationAccentColor() {
```

Comment:
Let's add the same verification for the other functions that are changing the notification colour like   `createDownloadGroupNotification, createPausedDownloadNotification,createDownloadFailedNotification,createDownloadCompletedNotification` :)

Reply:
I created others but unable to do `createDownloadGroupNotification` test. I can initialize 2 ongoing downloads and they'll be grouped but how can i get that group's color?